screens/main/home.js

```diff
@@ -232,6 +232,9 @@ export default function HomeScreen({navigation}) {
   const isToday = date.isSame(today);
   const dateString = isToday ? Strings.common.today : date.format('MMMM D');
 
+  const START_DATE = '2020-06-01';
+  const isBeforeStartDate = moment(today).isBefore(START_DATE);
```

Comment:
Not sure how these two lines of code got back into your branch...? The app is no longer be hard-coded to a specific start date....

Reply:
I'll remove it before merging in...

Reply:
I think I know what happened... you continued working on your same branch even after it was previously merged and deleted from a previous pull request... That's why this PR contains commits and changes that have already previously been merged...

Please always start by creating a new branch off of master...